Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -1581,6 +1581,61 @@
     tags:
     - Fruit # It's in the name
 
+- type: entity
+  name: capfruit
+  parent: FoodProduceBase
+  id: FoodRealCapfruit
+  description: A soft but smooth gun-shaped fruit.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - capfruit
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/capfruit.rsi
+  - type: Produce
+    seedId: realcapfruit
+  - type: Food
+    trash: RevolverCapGun
+  - type: Tag
+    tags:
+    - Fruit
+
+- type: entity
+  name: capfruit
+  parent: FoodProduceBase
+  id: FoodFakeCapfruit
+  suffix: Fake
+  description: A soft but smooth gun-shaped fruit.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - capfruit
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
+        - ReagentId: Sulfur
+          Quantity: 5
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/capfruit.rsi
+  - type: Produce
+    seedId: fakecapfruit
+  - type: Food
+    trash: RevolverCapGunFake
+  - type: Tag
+    tags:
+    - Fruit
```

Comment:
Think you can parent this with the real capfruit to remove some dupes. Like flavor and sprite (i think comment also parents?)

Reply:
only things that change are solution and trash